library/src/scala/quoted/Quotes.scala

```diff
@@ -3669,7 +3669,16 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
       */
       def newMethod(parent: Symbol, name: String, tpe: TypeRepr, flags: Flags, privateWithin: Symbol): Symbol
 
-      /** Generates a new val/var/lazy val symbol with the given parent, name and type.
+      /** Generates a new method symbol that overrides another definition.
+       *  The signature will be the same as the overridden symbol.
+       *
+       *  @param parent The owner of the method
+       *  @param overridden The symbol being overridden
+       */
+      @experimental
+      def newMethodOverride(parent: Symbol, overridden: Symbol): Symbol
```

Comment:
This API isn't flexible enough to handle situations where you need to override two methods at once. For example:
```scala
trait X; trait Y
trait A { def foo: X }
trait B { def foo: Y }
class C extends A with B {
  // ...
}
```
In C, if I try to override only `A#foo` or only `B#foo`, I will get a type error, I'm forced to override both at once.